tfjs-backend-webgpu/src/matmul_test.ts

```diff
@@ -877,4 +877,119 @@ describeWebGPU('matmul', () => {
+      74,  68,  17,  9,  21, 15, 8,  20,  29,  23,  27,  14, 11, 10, 8,  40,
+      144, 136, 85,  29, 45, 37, 22, 106, 135, 127, 87,  30, 43, 36, 22, 110,
+      139, 133, 92,  29, 44, 37, 22, 114, 144, 136, 85,  29, 45, 37, 22, 106,
+      45,  35,  50,  25, 18, 17, 14, 74,  90,  98,  93,  15, 29, 27, 14, 106,
+      131, 119, 25,  16, 37, 26, 14, 30,  45,  35,  50,  25, 18, 17, 14, 74,
+      95,  92,  117, 35, 35, 34, 23, 151, 184, 185, 109, 24, 55, 45, 23, 123,
+      158, 137, 50,  33, 48, 36, 23, 71,  95,  92,  117, 35, 35, 34, 23, 151,
+      31,  28,  29,  11, 11, 10, 7,  39,  48,  49,  37,  8,  15, 13, 7,  43,
+      54,  49,  18,  9,  16, 12, 7,  23,  31,  28,  29,  11, 11, 10, 7,  39,
+      47,  41,  23,  12, 15, 12, 8,  32,  41,  41,  42,  11, 14, 13, 8,  52,
+      64,  62,  31,  9,  19, 15, 8,  36,  47,  41,  23,  12, 15, 12, 8,  32,
+      29,  23,  27,  14, 11, 10, 8,  40,  49,  53,  52,  9,  16, 15, 8,  60,
+      74,  68,  17,  9,  21, 15, 8,  20,  29,  23,  27,  14, 11, 10, 8,  40,
+      144, 136, 85,  29, 45, 37, 22, 106, 135, 127, 87,  30, 43, 36, 22, 110,
+      139, 133, 92,  29, 44, 37, 22, 114, 144, 136, 85,  29, 45, 37, 22, 106,
+      45,  35,  50,  25, 18, 17, 14, 74,  90,  98,  93,  15, 29, 27, 14, 106,
+      131, 119, 25,  16, 37, 26, 14, 30,  45,  35,  50,  25, 18, 17, 14, 74,
+      95,  92,  117, 35, 35, 34, 23, 151, 184, 185, 109, 24, 55, 45, 23, 123,
+      158, 137, 50,  33, 48, 36, 23, 71,  95,  92,  117, 35, 35, 34, 23, 151,
+      31,  28,  29,  11, 11, 10, 7,  39,  48,  49,  37,  8,  15, 13, 7,  43,
+      54,  49,  18,  9,  16, 12, 7,  23,  31,  28,  29,  11, 11, 10, 7,  39,
+      47,  41,  23,  12, 15, 12, 8,  32,  41,  41,  42,  11, 14, 13, 8,  52,
+      64,  62,  31,  9,  19, 15, 8,  36,  47,  41,  23,  12, 15, 12, 8,  32,
+      29,  23,  27,  14, 11, 10, 8,  40,  49,  53,  52,  9,  16, 15, 8,  60,
+      74,  68,  17,  9,  21, 15, 8,  20,  29,  23,  27,  14, 11, 10, 8,  40,
+      144, 136, 85,  29, 45, 37, 22, 106, 135, 127, 87,  30, 43, 36, 22, 110,
+      139, 133, 92,  29, 44, 37, 22, 114, 144, 136, 85,  29, 45, 37, 22, 106,
+      45,  35,  50,  25, 18, 17, 14, 74,  90,  98,  93,  15, 29, 27, 14, 106,
+      131, 119, 25,  16, 37, 26, 14, 30,  45,  35,  50,  25, 18, 17, 14, 74,
+      95,  92,  117, 35, 35, 34, 23, 151, 184, 185, 109, 24, 55, 45, 23, 123,
+      158, 137, 50,  33, 48, 36, 23, 71,  95,  92,  117, 35, 35, 34, 23, 151,
+      31,  28,  29,  11, 11, 10, 7,  39,  48,  49,  37,  8,  15, 13, 7,  43,
+      54,  49,  18,  9,  16, 12, 7,  23,  31,  28,  29,  11, 11, 10, 7,  39,
+      47,  41,  23,  12, 15, 12, 8,  32,  41,  41,  42,  11, 14, 13, 8,  52,
+      64,  62,  31,  9,  19, 15, 8,  36,  47,  41,  23,  12, 15, 12, 8,  32,
+      29,  23,  27,  14, 11, 10, 8,  40,  49,  53,  52,  9,  16, 15, 8,  60,
+      74,  68,  17,  9,  21, 15, 8,  20,  29,  23,  27,  14, 11, 10, 8,  40
+    ]);
+  });
+
+  it('matMul MatMulPackedVec4Program outputShape[1] <= 1', async () => {
```

Comment:
nit: `outputShape[1] = 1`. Can `outputShape[1]` be less than 1?

Reply:
done